espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ActivityHelpers.kt

```diff
@@ -84,9 +88,18 @@ object ActivityHelpers {
                     "with default options: $defaultOptions")
             makeIntent(defaultOptions)
         } else {
-            AndroidLogger.logger.info("Staring activity with custom options: $intentOptions")
-            makeIntent(intentOptions)
+            AndroidLogger.logger.info("Staring activity with custom options: ${params.optionalIntentArguments}")
+            makeIntent(params.optionalIntentArguments)
+        }
+
+        makeActivityOptions(params.optionalActivityArguments).let {
```

Comment:
this won't work. There is no branch to execute if `params.optionalActivityArguments` is null

Reply:
if `params.optionalActivityArguments` is null, it will return default options 
`val options = ActivityOptions.makeBasic()` from makeActivityOptions.